validators.js

```diff
@@ -0,0 +1,29 @@
+const checkEmail = (email) => {
```

Comment:
이메일의 무엇을 확인하는지 조금 애매한 함수명 같습니다.
이메일 데이터에 대해 형식이 올바른지 유효성검사를 하는 함수이기 때문에 다음과 같이 수정해주세요.
```suggestion
const validateEmail = (email) => {
```